src/lib/encode-conversions.ts

```diff
@@ -105,13 +105,21 @@ export const convertObservation: ConvertFunction<'observation'> = (
   const metadata: Observation_1_Metadata = mapeoDoc.metadata && {
     ...Observation_1_Metadata.fromPartial(mapeoDoc.metadata),
   }
+  let presetRef
+  if (mapeoDoc.presetRef) {
+    presetRef = {
+      docId: Buffer.from(mapeoDoc.presetRef?.docId, 'hex'),
+      versionId: parseVersionId(mapeoDoc.presetRef?.versionId),
```

Comment:
The nullish check is unnecessary, and if it was it would result in a runtime error (`Buffer.from(undefined, 'hex')` throws an error).

```suggestion
      docId: Buffer.from(mapeoDoc.presetRef.docId, 'hex'),
      versionId: parseVersionId(mapeoDoc.presetRef.versionId),
```

Reply:
oh yeah, I forgot to remove it after adding the conditional. Thanks for the catch, merging now